src/amo/components/ServerHtml/index.js

```diff
@@ -110,6 +120,21 @@ export default class ServerHtml extends Component {
     )}`;
   }
 
+  getFontPreload() {
+    const { assets, htmlLang } = this.props;
+    // Preload relevant minimal font if available. At the moment only for en-US.
+    if (htmlLang === 'en-US') {
+      const fontToPreload =
+        './src/fonts/woff2/Inter-roman-minimal-subset.var.woff2';
```

Comment:
Obviously hardcoding the name here isn't great. Thoughts on where/how to do that ? Note that it will be slightly dynamic when the same principle is applied to other locales as well.

Reply:
You could iterate over `Object.keys(assets.assets)` and filter out everything not in a `/fonts/` folder and containing `subset` maybe?

Reply:
I like where this is going... I'd need to generate the subset with the `htmlLang` in it too to be future-proof... Check with a regexp... And then, in the near future, when we start doing it for other languages, it will work right away. And we could do something like call a subset `Inter-roman-minimal-subset-fr-it-es.var.woff2` to support multiple languages in a single subset.